compiler/luci/service/src/Nodes/CircleOutput.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <luci/Service/CircleShapeSignatureInferenceRule.h>
+
+namespace luci
+{
+
+ShapeSignature ShapeSignatureInferenceAlgorithm::visit(const luci::CircleOutput *node)
```

Comment:
Is there any particular reason to put one op each files?

Reply:
Yes, final goal is to put implementations of `ShapeInferenceAlgorithm`, `TypeInferenceAlgorithm`, `ShapeSignatureInferenceAlgorithm` for each nodes in one file.
Here is [an example](https://github.com/Samsung/ONE/blob/245c0b3c68a7f18e259582126b8f88bfb35fe632/compiler/luci/service/src/Nodes/CircleRelu.cpp) in draft code :)

Reply:
But wuy?